### ISSUE TITLE
Town of Amherst, MA

### ISSUE THREAD

**Issue by justinelliotmeyers** [openaddresses/openaddresses#3581]
http://gis.amherstma.gov/arcgis/rest/services/gpv/GpvParcels/MapServer/2

**Comment by eugeneYWang** [comment]
MA has a great GIS department MASSGIS that handles statewide address point. So no need to worry about town-level data in MA

**Comment by ingalls** [comment]
@eugeneYWang When we do have a statewide source we just don't go out of our way to track down duplicative source but we do track as many address files as possible as these often update more frequently than the statewide sources.

**Comment by eugeneYWang** [comment]
@ingalls Thank you for your explanation. I have one question for OpenAddress: is all the US lat/long under WGS84 even though original data are under multiple Projections?

**Comment by ingalls** [comment]
@eugeneYWang Yes - All coordinates in our output data should be WGS84 (EPSG:4326).

Occasionally you may find a source where this is not the case and it is usually due to the provider not giving us projection info. Feel free to open an issue and tag me and we will get it fixed.

The same is true for the number/street fields. Occasionally you may find sources that have been parsed into the wrong fields as the provider changed their underlying data or our parsing code doesn't handle an edge case. Same as above we would love to know about it and fix it so just open an issue and we'll get it fixed.
